packages/components/view-switcher/src/view-switcher.spec.js

```diff
@@ -0,0 +1,148 @@
+import { warning } from '@commercetools-uikit/utils';
```

Comment:
Let me pin it here. I am wondering if visual regression tests should also be part of this PR